cli/fluffy_cli/main.py

```diff
@@ -29,88 +29,92 @@
     {"server": "https://fluffy.my.corp"}
 
 This file can be placed at either /etc/fluffy.json or ~/.config/fluffy.json.
-'''
+"""
 
 
 def bold(text):
     if sys.stdout.isatty():
-        return '\033[1m{}\033[0m'.format(text)
+        return "\033[1m{}\033[0m".format(text)
     else:
         return text
 
 
 def get_config():
-    config = {'server': 'https://fluffy.cc'}
-    for path in ('/etc/fluffy.json', os.path.expanduser('~/.config/fluffy.json')):
+    config = {"server": "https://fluffy.cc"}
+    for path in ("/etc/fluffy.json", os.path.expanduser("~/.config/fluffy.json")):
         try:
             with open(path) as f:
                 j = json.load(f)
                 if not isinstance(j, dict):
                     raise ValueError(
-                        'Expected to parse dict, but the JSON was type "{}" instead.'.format(type(j)),
+                        'Expected to parse dict, but the JSON was type "{}" instead.'.format(
+                            type(j)
+                        )
                     )
                 for key, value in j.items():
                     config[key] = value
         except FileNotFoundError:
             pass
         except Exception:
-            print(bold('Error parsing config file "{}". Is it valid JSON?'.format(path)))
+            print(
+                bold('Error parsing config file "{}". Is it valid JSON?'.format(path))
+            )
             raise
     return config
 
 
 def upload(server, paths, auth, direct_link):
-    files = (('file', sys.stdin.buffer if path == '-' else open(path, 'rb')) for path in paths)
+    files = (
+        ("file", sys.stdin.buffer if path == "-" else open(path, "rb"))
+        for path in paths
+    )
     req = requests.post(
-        server + '/upload?json',
-        files=files,
-        allow_redirects=False,
-        auth=auth,
+        server + "/upload?json", files=files, allow_redirects=False, auth=auth
```

Comment:
don't like this at all... hmm

why isn't it at least adding trailing commas if it's going to do this?